app/src/main/java/org/oppia/android/app/settings/profile/ProfileEditActivityPresenter.kt

```diff
@@ -38,9 +34,9 @@ class ProfileEditActivityPresenter @Inject constructor(
     )
     val profileId = activity.intent.getIntExtra(KEY_PROFILE_EDIT_PROFILE_ID, 0)
     editViewModel.setProfileId(
-      profileId,
-      activity.findViewById<Switch>(R.id.profile_edit_allow_download_switch)
+      profileId
```

Comment:
`editViewModel.setProfileId(profileId)` it should be in single line

Reply:
DONE